signalfx_lambda/metrics.py

```diff
@@ -123,8 +123,14 @@ def wrapper(*args, **kwargs):
             default_dimensions.update(dimensions)
 
         token = kwargs.get('access_token')
-        if isinstance(token, basestring):
+        if isstr(token):
```

Comment:
Since you already depend on `six`, a more common way of doing this check is:

```python
import six
...
if isinstance(token, six.string_types):
    access_token = token
```